Live Testing update - App Management (#1684)
saucelabs/sauce-docs: docs/mobile-apps/live-testing/live-mobile-app-testing.md

To upload an app via the Sauce Labs UI:

1. On Sauce Labs, in the left panel, click **App Management**.

> 2. To upload an app you can either drag and drop an app or browse for and select the file. We currently support \*.apk Android app files, \*.aab Android App Bundle files and \*.ipa or \*.zip iOS app files (\*.zip files are parsed to determine whether a valid \*.app bundle exists). Non-app file uploads are not supported in the UI at this time, but can be uploaded through the API.

Reviewer: ```suggestion
2. To upload an app you can either drag and drop an app or browse for and select the file. We currently support \*.apk Android app files, \*.aab Android App Bundle files, and \*.ipa or \*.zip iOS app files (\*.zip files are parsed to determine whether a valid \*.app bundle exists). Non-app file uploads are not supported in the UI at this time, but can be uploaded through the API.
```

If you don't have an app to test, you can use the [Sauce Labs sample mobile app](https://github.com/saucelabs/sample-app-mobile).

<img src={useBaseUrl('img/live-testing/live-mobile-app-management.png')} alt="Upload an app" width="650"/>

### Deleting an App

Deleting an app in Sauce Labs will delete the whole app (i.e., the group of builds belonging to the same app package). Files associated with app identifiers (i.e., belonging to the same platform and accessible to the same team) are indicated by the + symbol next to the version number. Also, the version number shown is the most recently updated file, not necessarily the latest version of the app.

> To delete an app, on the App Management page, hover over the build and then click **Delete**.

Reviewer 2: ```suggestion
To delete an app, on the **App Management** page, hover over the build and then click **Delete**.
```

> To delete an app, on the **App Management** page, hover over the build and then click **Delete**.

Reviewer: ```suggestion
To delete an app, on the **App Management** page, hover over the app and then click **Delete**.
```

<img src={useBaseUrl('img/live-testing/live-mobile-app-management-delete.png')} alt="Delete an app" width="850"/>

### App Settings

> To view or change the app settings, on the App Management test page, hover over the app and then click **Settings**.

Reviewer 2: ```suggestion
To view or change the app settings, on the **App Management** page, hover over the app and then click **Settings**.
```

Reviewer 2: The step says to click "Settings" but the tooltip says "Settings and App Versions"

| Setting | Description |
| :--- | :--- |

> | Device Language | Use the dropdown to select the device language. The language selector will tell your application that the locale of the device and region is set to the selected parameter. You won't need to change the language of the OS manually during a session inside iOS/Android settings. ([Read more about Locale here](https://developer.apple.com/documentation/foundation/locale)).  |

Reviewer 2: ```suggestion
| Device Language | Use the dropdown to select the device language. The language selector will tell your application that the locale of the device and region is set to the selected parameter. You won't need to change the language of the OS manually during a session inside iOS/Android settings. For more information about the locale setting, see the documentation for [iOS](https://developer.apple.com/documentation/foundation/locale) and [Android](https://developer.android.com/reference/java/util/Locale).  |
```

**Example Settings - iOS**

<img src={useBaseUrl('img/live-testing/live-mobile-app-management-ios.png')} alt="App settings - iOS" width="750"/>

**Example Settings - Android**

<img src={useBaseUrl('img/live-testing/live-mobile-app-management-android.png')} alt="App settings - Android" width="780"/>

Most settings update automatically, however, when you make changes to the proxy setting, click **Update** to finish.

### Selecting a Device

You must select a device prior to launching a session.

On the **App Management** page, hover over the app you want to test and then click **Start Test**.

<img src={useBaseUrl('img/live-testing/live-mobile-app-management-test.png')} alt="Choose a device" width="750"/>

The device selection page will open, with the option to test on a real device or a virtual device.

:::note

> If you are testing an IOS app, the device selection will only display the type (real/virtual) configured on the app.

Reviewer 2: ```suggestion
If you are testing an iOS app, the device selection will only display the type (real/virtual) configured on the app.
```

| <img src={useBaseUrl('img/live-testing/rotate-device-icon.png')} alt="Rotate Device icon" width="35"/>             | Rotate Device       | Rotates the device between portrait and landscape.                                                                                                                                                                                                                                                                                            |
| <img src={useBaseUrl('img/live-testing/home-icon.png')} alt="Home icon" width="35"/>                               | Home                | Opens the device home screen.                                                                                                                                                                                                                                                                                                                 |

> | <img src={useBaseUrl('img/live-testing/more-device-options-icon.png')} alt="More Device Options icon" width="35"/> | More Device Options | **Set Location** - Set the GPS location using coordinates or by dropping a pin on the map. <br/> **Camera Injection** - Opens the **Camera Injection** window. See [Camera Image Injection](/mobile-apps/features/camera-image-injection) for more information.<br/> **Biometric Injection** - Opens the [Biometric Authentication](https://docs.saucelabs.com/mobile-apps/features/biometric-authentication/) window. |

Reviewer 2: ```suggestion
| <img src={useBaseUrl('img/live-testing/more-device-options-icon.png')} alt="More Device Options icon" width="35"/> | More Device Options | **Set Location** - Set the GPS location using coordinates or by dropping a pin on the map. <br/> **Camera Injection** - Opens the **Camera Injection** window. See [Camera Image Injection](/mobile-apps/features/camera-image-injection) for more information.<br/> **Biometric Injection** - Opens the [Biometric Authentication](/mobile-apps/features/biometric-authentication) window. |
```
